### DISCUSSION TITLE
Aircraft state in EFB/MCDU

### DISCUSSION THREAD

**Discussion by DrFilippas** [flybywiresim/aircraft#6606] [Ideas]
Hi everyone,

Any chance we could get an option to set the **aircraft's state** (Cold & Dark, Turnaround, Ready to Taxi, Ready for Take off etc) in the **EFB** or **MCDU**?
I've seen some relevant posts about it in the past but no development ever since.

Keep up the good work! 
Many thanks.

**Comment by tracernz** [comment]
We would like to do this eventually, but it's lower priority than implementing the systems it will actually initialize.

**Reply by DrFilippas** [reply]
Thanks for the reply. Looking forward to it!

Is there any way I could save the cockpit state at least?
